### ISSUE TITLE
introduce shorthand URL for `$projectResources"

### ISSUE THREAD

**Issue by dariok** [dariok/wdbplus#469]
Similar to the general $shared, we need a similar construct for projectResource, e.g.:

`$r-{$projectID}` → `{$wdb:data}/{$projectCollection}/resources`

`$projectCollection` must be obtained from `wdbmeta.xml` identified by `$projectID`

**Comment by dariok** [comment]
While `$global` is straightforward, `$r-{$projectID}` can (and should) be part of additional changes:

If we resolve this in controller.xql, we need to know `$projectResources`, which we can get from a model. But this would mean to `populateModel()` in `controller.xql` and again when actually showing a page.

It is a lot easier to `populateModel()` only once – i.e., in `controller.xql` – and then pass it on to subsequent functions as part of a parameter. This should also take care of #601.

Will have to postpone this second part, though, as APIv2 has priority right now.
